### ISSUE TITLE
Mod Picker: Don't allow selecting multiple of unstackable mods

### ISSUE THREAD

**Issue by bhollis** [DestinyItemManager/DIM#10096]
We should detect cases like these and not allow selecting more than one:
<img width="373" alt="Screenshot 2023-11-11 at 2 31 34 PM" src="https://github.com/DestinyItemManager/DIM/assets/313208/0d3df93e-ac37-4516-884b-6497b7c3edf5">

We can also maybe replace this verbose message with "Stackable" and "Unstackable" labels.

**Comment by grSalonga** [comment]
How would you check if a mod is stackable? From what I can see, I don't think the dim api has an identifier for this.

**Comment by bhollis** [comment]
No, it wouldn't. You'd have to compute a mapping in d2ai and use that.

**Comment by grSalonga** [comment]
So that would be creating a "generate-unstackable-mods.ts" file in d2-additional-info that would output a json file that contains all hashes of mods that don't stack?

**Comment by bhollis** [comment]
Yeah that's probably the best way to do it.

**Comment by grSalonga** [comment]
I made a generate file that grabs the hashes for mods that don't stack. The only issue I'm having now is that I cant get ```yarn generate-data``` to execute the js file. I added a generate line to the ```package.json``` file and it is creating the js file in the built folder. When I run the js file by itself it is generating the output file in the output folder not in built.  What else am I missing?
